CacheManager.StackExchange.Redis.SelfCleaning.Tests/DummyModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using ProtoBuf;
+
+namespace CacheManager.StackExchange.Redis.SelfCleaning.Tests
+{
+    [ProtoContract]
+    public class DummyModel
+    {
+        [ProtoMember(1)]
```

Comment:
This is only because I use Protobuf serialization when building the cache (there must be some kind of `ISerializer` otherwise it will throw an exception). Technically, I can use any other kind of serialization, such as JSON, or even pass a dummy `ISerializer` that I build (although that might be a bit complex since once again CacheManager uses reflection to construct the serializer).

About the other questions:
1. It specifies which members will be included in the contract, and in what order.
2. The attributes don't really affect the class itself, so I don't see why.
3. I think that depends on the Protobuf serializer and the way it is implemented in CacheManager. As far as I've seen, if these attributes aren't in place the serialized object will be empty. There might be a workaround though.

Reply:
OK, cool.
The JSON serializer require attributes too? if not, we might want to use it rather the protobuff (just because it slightly less clear when you look at the model).

You it does, so let stick with the protobuff, and maybe add comment above Contract attribute: "Required to support ProtobuffSerializer which being used in integration tests"

Reply:
JSON serialization doesn't require attributes, changed to that